config/routes.rb

```diff
@@ -2,6 +2,8 @@
   # get 'dashboards/index'
   root 'dashboards#index'
 
+get '/food/:id', to: 'food#show'
```

Comment:
we do not need this

Reply:
@Tresor11  Thanks so much for the review , I'm going make changes as required .